plugins/inputs/elasticsearch_query/elasticsearch_query.go

```diff
@@ -0,0 +1,277 @@
+package elasticsearch_query
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	elastic "gopkg.in/olivere/elastic.v5"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/internal/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const description = `Queries Elasticseach`
+const sampleConfig = `
+  ## The full HTTP endpoint URL for your Elasticsearch instance
+  ## Multiple urls can be specified as part of the same cluster,
+  ## this means that only ONE of the urls will be written to each interval.
+  urls = [ "http://node1.es.example.com:9200" ] # required.
+  ## Elasticsearch client timeout, defaults to "5s" if not set.
+  timeout = "5s"
+  ## Set to true to ask Elasticsearch a list of all cluster nodes,
+  ## thus it is not necessary to list all nodes in the urls config option
+  enable_sniffer = false
```

Comment:
Maybe `cluster_node_discovery`? We do not really sniff do we?

Reply:
This setting is coming from the client: https://github.com/olivere/elastic/wiki/Sniffing
I'd be fine by removing or hiding this (and health check).

Reply:
Yeah I'm only asking because the name is not very self-containing and it's barely expresses what it does. Furthermore, sniffing might have a negative co-notation... But if you want to keep it, it's up to you.